src/main/java/it/gov/pagopa/bizeventsdatastore/BizEventToDataStore.java

```diff
@@ -33,7 +33,7 @@ public void processBizEvent (
             @CosmosDBOutput(
     	            name = "BizEventDatastore",
     	            databaseName = "db",
-    	            collectionName = "biz-events",
+    	            collectionName = "biz-events-test",
```

Comment:
it must be `biz-events` because it is defined at `pagopa-infra` level.
For test purposes, you could set the collection name in `application.properties`, `createIfNotExists` too which is disabled by default.

Reply:
It seemed to me more straightforward do this by updating the local settings already used.
Concerning `createIfNotExists` I am looking for a way to read a boolean (via local settings) and pass it directly as a parameter, but doesn't seem to be one.